### PR TITLE
add aquachain node

### PR DESCRIPTION
**Pull request by aerth**
Adds initial support for Aquachain

[Website](https://aquachain.github.io)
[Explorer](https://aquachain.github.io/explorer/)
[Source](https://gitlab.com/aquachain/aquachain)
[Releases](https://github.com/aquanetwork/aquachain/releases)
[Bitcointalk](https://bitcointalk.org/index.php?topic=3138231)

Chain ID: 61717561
POW Algorithm: argon2id (+ASIC resistant)

### CODE REVIEW

**Comment by aerth** [comment]
Exposes issue with chain id limitations as seen https://github.com/trezor/connect/issues/137

**Comment by aerth** [comment]
PTAL

**Comment by Azarielle** [comment]
[Added to Asana](https://app.asana.com/0/763966770926912/779983315837073)

**Comment by aerth** [comment]
i thought the failing test was inevitable when adding a new network, if that is not the case, i'd be happy to try and get it passing.

File: common/config/dpaths.ts [hunk 1]
```diff
@@ -135,7 +140,8 @@ export const DPaths: DPath[] = [
   RSK_TESTNET,
   GO_DEFAULT,
   EOSC_DEFAULT,
-  ESN_DEFAULT
+  ESN_DEFAULT,
```
Comment (aerth) [inline common/config/dpaths.ts]: comma

**Comment by aerth** [comment]
OK it seems like some commas were out of place after a squash or something, fixing

**Comment by aerth** [comment]
Test now passes

**Pull request merged**